wren-ai-service/src/pipelines/generation/relationship_recommendation.py

```diff
@@ -54,8 +55,28 @@ def validated(normalized: dict, engine: Engine) -> dict:
 
 
 ## End of Pipeline
+class ModelRelationship(BaseModel):
+    name: str
+    models: list[str]
+    joinType: str
+    condition: str
+    reason: str
 
-system_prompt = """ 
+
+class RelationshipResult(BaseModel):
+    relationships: list[ModelRelationship]
+
+
+Relationship_MODEL_KWARGS = {
```

Comment:
could you also make this capitalized? thanks! @MGomaa435

Reply:
done